docs/setting-up/expo.md

```diff
@@ -80,3 +80,6 @@ Next, rebuild your app and you're good to go!
 ```sh
 npx expo run:android && npx expo run:ios
 ```
+
+Now you're ready to [set up your configuration files](/docs/setting-up/get-config-file).
+
```

Comment:
```suggestion
```

If you read the expo guide carefully, you'll notice there are 2 references to the "get config" guide. Referring to the guide here is too late in the process.